ios/RNPurchases.m

```diff
@@ -35,13 +35,14 @@ - (dispatch_queue_t)methodQueue
 RCT_EXPORT_METHOD(setupPurchases:(NSString *)apiKey
                   appUserID:(NSString *)appUserID
                   observerMode:(BOOL)observerMode
+                  userDefaultsSuiteName:(NSString *)userDefaultsSuiteName
```

Comment:
does this get automatically translated to `nil` when it's not set? 
I'm asking because in flutter it was getting translated to `NSNull`, so we had to do some extra conversion. 
If it is nil, maybe it's worth adding `nullable` here?

Reply:
It does get translated to nil, I will add the `nullable` :)